layout/head/app/component.js

```diff
@@ -98,14 +98,13 @@ class HeadApp extends PureComponent {
     return (
       <HeadNext>
         <title>{title ? `${title} | Resource Watch` : 'Resource Watch'}</title>
+
+        <meta property="og:image" content="https://resourcewatch.org/static/images/social-big.jpg" />
```

Comment:
`og:image` and `og:image:url` are equals, see:
![image](https://user-images.githubusercontent.com/999124/55222867-3cd2a100-520d-11e9-8510-fedc49d8ff2c.png)

Remove one of them and make the remaining one variable.